lib/OrdinaryDiffEqExponentialRK/src/algorithms.jl

```diff
@@ -37,12 +79,37 @@ for Alg in [:Exprb32, :Exprb43]
             iop)
     end
 end
-for Alg in [:Exp4, :EPIRK4s3A, :EPIRK4s3B, :EPIRK5s3, :EXPRB53s3, :EPIRK5P1, :EPIRK5P2]
-    @eval struct $Alg{CS, AD, FDT, ST, CJ} <:
-                 OrdinaryDiffEqExponentialAlgorithm{CS, AD, FDT, ST, CJ}
-        adaptive_krylov::Bool
-        m::Int
-        iop::Int
+for (Alg, Description, Ref) in [
+    (:Exp4, "4th order EPIRK scheme.", "REF TBD")
+    (:EPIRK4s3A, "4th order EPIRK scheme with stiff order 4.", "REF TBD")
+    (:EPIRK4s3B, "4th order EPIRK scheme with stiff order 4.", "REF TBD")
+    (:EPIRK5s3, "5th order “horizontal” EPIRK scheme with stiff order 5. Broken.", "REF TBD")
+    (:EXPRB53s3, "5th order EPIRK scheme with stiff order 5.", "REF TBD")
+    (:EPIRK5P1, "5th order EPIRK scheme", "REF TBD")
+    (:EPIRK5P2, "5th order EPIRK scheme", "REF TBD")
```

Comment:
@ChrisRackauckas refs

Reply:
https://github.com/SciML/OrdinaryDiffEq.jl/issues/394 has the complete set of references.